cmd/restic/global.go

```diff
@@ -95,6 +97,14 @@ func init() {
 		return nil
 	})
 
+	//set MinPackSize to 4 if not set in env
+	minPackSize, err := strconv.Atoi(os.Getenv("RESTIC_MIN_PACKSIZE"))
+	if err != nil {
+		minPackSize = 4
+	} else if minPackSize < 1 {
+		minPackSize = 1
```

Comment:
maybe error or warn the user that we're ignoring the provided value?

Reply:
Agreed,  we definitely should.  

Something like "pack size must be a positive, non-zero integer"

And it might actually make sense to default to 4 when we're out of range, now that I think about it.